dev/TreeView/TestUI/TreeViewLateDataInitTestPage.xaml.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System;
+using Windows.UI.Xaml;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+
+namespace MUXControlsTestApp
+{
+    /// Test page for late initialization data binding. ItemsSource is null until we click the button to set it.
+    public sealed partial class TreeViewLateDataInitTest : TestPage, INotifyPropertyChanged
+    {
+        public event PropertyChangedEventHandler PropertyChanged;
+        private ObservableCollection<TreeViewItemSource> m_testTreeViewItemsSource;
+        public ObservableCollection<TreeViewItemSource> TestTreeViewItemsSource
+        {
+            get { return m_testTreeViewItemsSource; }
+            set
+            {
+                if (m_testTreeViewItemsSource != value)
+                {
+                    m_testTreeViewItemsSource = value;
+                    NotifyPropertyChanged("TestTreeViewItemsSource");
```

Comment:
You can use nameof here to make things a bit more strongly typed.

Reply:
I got an error saying nameof operator is a C# 6 feature, we are currently on 5, are we ok upgrading to 6?

Reply:
@llongley hit some issues there. Would be nice to move to the latest version.

Reply:
I hit this as well in late October. There wasn't a reason to not do it anymore assuming the OS build still functions (and it'll noisily fail if it isn't supported). Edit the LangVersion to give it a try if you like.

Reply:
I updated to LangVersion to 7 and manually queued an os build https://microsoft.visualstudio.com/WinUI/WinUI%20Team/_build/results?buildId=13278936. Seems fine to me?